logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/conversation/MLSWelcomeEventHandler.kt

```diff
@@ -57,29 +57,26 @@ internal class MLSWelcomeEventHandlerImpl(
                     client.processWelcomeMessage(event.message.decodeBase64Bytes())
                 }
             }.flatMap { groupID ->
-                val groupIdLogPair = Pair("groupId", groupID.obfuscateId())
-
                 conversationRepository.fetchConversationIfUnknown(event.conversationId)
                     .flatMap {
                         markConversationAsEstablished(GroupID(groupID))
                     }.flatMap {
                         resolveConversationIfOneOnOne(event.conversationId)
-                    }.onSuccess {
-                        kaliumLogger
-                            .logEventProcessing(
-                                EventLoggingStatus.SUCCESS,
-                                event,
-                                Pair("info", "Established mls conversation from welcome message"),
-                                groupIdLogPair
-                            )
-                    }.onFailure {
-                        kaliumLogger
-                            .logEventProcessing(
-                                EventLoggingStatus.FAILURE,
-                                event,
-                                groupIdLogPair
-                        )
-                }
+                    }
+            }.onSuccess {
```

Comment:
Moved it out because we didn't log anything if `client.processWelcomeMessage(event.message.decodeBase64Bytes())` did throw an error.